web/src/pages/Namespace.tsx

```diff
@@ -101,6 +101,20 @@ export const NamespacePage: FC = () => {
         <NamespaceInfoPlaceholder />
       ) : (
         <>
+          { namespace === user?.login && user?.orgs && user.orgs.length > 0 && (
+          <p className="mb-0">
+            <span className="fw-bold">Organizations you belong to: {' '}
+              {user?.orgs.map((org, index) => (
+                <Fragment key={org}>
+                  {index !== 0 && ', '}
```

Comment:
Curious what the `index !== 0` means here? Why skip the first? Is it because it's their username?

Reply:
If it's not equal to 0 we can render a comma before the next organization name. There's prob a better way to do this but it works for not rendering a comma first and in the last element

Reply:
Oh I see. That makes sense - good solution